mocked-api/handlers.ts

```diff
@@ -180,36 +210,278 @@ export const handlers = [
 
     return res(ctx.status(202), ctx.delay(apiDelay), ctx.json({}));
   }),
+
+  // processors
+  // get all processors of a bridge
+  rest.get(`${apiUrl}/bridges/:bridgeId/processors`, (req, res, ctx) => {
+    const { bridgeId } = req.params;
+
+    const page = parseInt(req.url.searchParams.get("page") ?? "0");
+    const size = parseInt(req.url.searchParams.get("size") ?? "10");
+
+    const query = {
+      where: {
+        bridge: {
+          id: {
+            equals: bridgeId as string,
+          },
+        },
+      },
+    };
+
+    const count = db.processor.count(query);
+
+    return res(
+      ctx.status(200),
+      ctx.delay(apiDelay),
+      ctx.json({
+        kind: "ProcessorList",
+        items: db.processor
+          .findMany({
+            take: size,
+            skip: page * size,
+            orderBy: {
+              submitted_at: "desc",
+            },
+            ...query,
+          })
+          .map((item) =>
+            cleanupProcessor(
+              item as unknown as Record<string | number | symbol, unknown>
+            )
+          ),
+        page: page,
+        size: size,
+        total: count,
+      })
+    );
+  }),
+  // get a single processor
```

Comment:
Here be more dragons, uglier and nastier than the first.. 

The real backend will return `ItemNotFoundException` (i.e. code 4) if the `Bridge` or the `Processor` does not exist.

I think we should create a JIRA to differentiate between the two in the backend: `BridgeNotFound` and `ProcessorNotFound`. @robpblake WDYT? At the moment a call from the UI to get Processors for a Bridge could return `ItemNotFoundException` for either the `Bridge` or `Processor` not being found.

It is a scenario that should not exist (at least within the UI as it will only list Bridges that do exist and the User will only be able to navigate to a Processor for an existing `Bridge` however it remains a valid concern for the REST API).